go.mod

```diff
@@ -12,24 +12,24 @@ require (
 	github.com/pires/go-proxyproto v0.6.2
 	github.com/prometheus/client_golang v1.17.0
 	github.com/shirou/gopsutil v3.21.11+incompatible
-	golang.org/x/net v0.24.0
-	golang.org/x/sync v0.7.0
-	golang.org/x/sys v0.19.0
+	golang.org/x/net v0.33.0
+	golang.org/x/sync v0.10.0
+	golang.org/x/sys v0.28.0
 	golang.org/x/time v0.5.0
-	google.golang.org/grpc v1.64.0
-	google.golang.org/protobuf v1.34.1
```

Comment:
Just to note, the profobuf and grpc versions here were kept at the same version in our github actions and python install script.

It's probably not required to do so though.